src/content/1.7/testing/web-acceptance-tests/how-to-contribute-and-create-web-acceptance-tests.md

```diff
@@ -0,0 +1,213 @@
+
+When a test is finished, all created/downloaded files are deleted using a function from the same file : ‘deleteFile` as part of the cleaning behind approach.
+
+#### Initialize pages
+In each and every test, we initialize the pages that will be needed. The initialization is done in a function called `init` which returns an object with multiple entries (pages) of pages.
+
+This function is called at the beginning of the test and each time we change the browser’s tab.
+
+Example: 
+```
+// For test ‘Filter Customer’
+const init = async function () {
+  return {
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    customersPage: new CustomersPage(page),
+  };
+};
+```
+  
+#### Expect
+We use the `expect` keyword from the `chai` library (https://www.chaijs.com/api/bdd/). This allows us to write assertions in a much more readable way. You can use whatever way to assert you want/need. Don’t forget that you can use the second argument of `expect` to log out a better error message when your assertion fails.
+
+Example :
+
+``` 
+Const isCustomerConnected = await this.pageObject.foLoginPage.isCustomerConnected();
+await expect(isCustomerConnected, ‘Customer is disconnected in FO’).to.be.true;
+```  
+
+#### Test identifier
+Our team thinks it’s very important to be able to follow how tests results evolve. So it’s been decided to add a unique identifier to every step in the test.
+
+Why? To be able to identify each and every test and compare from one report to another, to know how the results change.
+
+For example, you could have 10 failed tests one day, and 10 the day after. How do you know if it’s the same 10 tests failing, or another distribution (for example, 5 tests fixed, 5 other tests failing)? With this system we can calculate the evolution and show it on the Board.
+
+For that, we first created a base context for each and every test file, and after that we make a call to the function `addContextItem` with the unique value inside the test file. 
+
+Example :
+
+```
+// From test : puppeteer/campaigns/functional/BO/04_customers/01_customers/07_helpCard.js
+const baseContext = 'functional_BO_customers_customers_helpCard';
+// And inside each `it`, we make a call
+// For example, in the “Go To Customer’s Page” step we will have :
+testContext.addContextItem(this, 'testIdentifier', 'goToCustomersPage', baseContext);
+// In the report, the final identifier will look like this: ‘functional_BO_customers_customers_filterAndQuickEditCustomers_goToCustomersPage’
+```
+
+Be careful, identifiers **must be unique** through the whole campaign !
+
+#### Cleaning behind
+You must be able to launch a test independently, as well as in a whole campaign. That means :
+
+- Your test must create its own data or rely on the default fixtures
+- Your test must clean behind itself in a reliable manner
+- Deleting files (invoices, images) and artifacts
```

Comment:
```suggestion
* Your test must create its own data or rely on the default fixtures
* Your test must clean behind itself in a reliable manner
* Deleting files (invoices, images) and artifacts
```

Reply:
I don't understand this suggestion, it displays like this 
![image](https://user-images.githubusercontent.com/48441421/79586996-2b875b80-80d2-11ea-8d7e-6e199c32c2dd.png)